### ISSUE TITLE
Leader board should show the total steps 

### ISSUE THREAD

**Issue by AnnaDodson** [AnnaDodson/step-challenge#67]
It would be nice if the total steps of everyone was displayed on the leaderboard

**Comment by daniel-winfield** [comment]
Has this been done?

**Comment by AnnaDodson** [comment]
Nope! Want to pick it up? Could either be a JavaScript count on the front end or a new GraphQL query to get the total

**Comment by AnnaDodson** [comment]
Added in PR #76 

Could also include the total in the leader board when step counts are hidden

**Comment by daniel-winfield** [comment]
Yes I think that would be a good improvement
